test/common-utils/scenarios.json

```
@@ -126,6 +126,16 @@
             }
         }
     },
+    "configure_zsh_no_template_first_step": {
+        "image": "debian:bullseye",
+        "remoteUser": "vscode",
```

```suggestion
        "remoteUser": "devcontainer",
```

I wonder if this would fix https://github.com/devcontainers/features/actions/runs/5849488185/job/15890491045?pr=653#step:4:15987